mlflow/spark.py

```diff
@@ -76,7 +76,7 @@
 FLAVOR_NAME = "spark"
 
 # Default temporary directory on DFS. Used to write / read from Spark ML models.
-DFS_TMP = "/tmp/mlflow"
+DFS_TMP = os.environ.get("MLFLOW_DFS_TMP", "/tmp/mlflow")
```

Comment:
Can we define this environment variable in `mlflow.environment_variables` for documentation purposes? https://mlflow.org/docs/latest/python_api/mlflow.environment_variables.html

Reply:
Sure!